extensions/wikia/ImageReview/modules/PromoteImage/PromoteImageReviewSpecialController.class.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
-class PromoteImageReviewSpecialController extends ImageReviewSpecialController {
-	const DEFAUL_IMAGE_SIZE = 320;
+// TODO make a note about how this is completely fubared and we don't care about it anymore, but how it's
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)